scripts/sync_makefiles.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# GITHUB_TOKEN required scope: repo.repo_public
+
+git_mail="prombot@gmail.com"
```

Comment:
Ping @krasi-georgiev. Is there an email address associated to the prombot account?

Reply:
prometheus-team@googlegroups.com